make/libraries

```diff
@@ -33,6 +33,10 @@ $(CVODES)/lib/libsundials_nvecserial.a: $(SUNDIALS_NVECSERIAL)
 
 LIBCVODES = $(CVODES)/lib/libsundials_nvecserial.a $(CVODES)/lib/libsundials_cvodes.a
 
+STAN_CVODES_HEADERS := $(shell find stan -name *cvodes*.hpp)
```

Comment:
Yes, this works on Windows when RTools is installed. RTools installs `find`.

Re: Python, are you talking PyStan? I still haven't fully figured out PyStan's build process, but some of it will be independent of this.